clients/samba/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG baseimage=meldsun/samba
+ARG tag=latest
+FROM $baseimage:$tag
+RUN echo "latest" > /version.txt
+
+ENTRYPOINT ["samba-entry.sh"]
+
+
+USER root
+
+ADD samba.sh /opt/samba/bin/samba.sh
+RUN chmod +x /opt/samba/bin//samba.sh
+ENTRYPOINT ["/opt/samba/bin/samba.sh"]
```

Comment:
please add a new line at the bottom of the file to remove 
![image](https://github.com/user-attachments/assets/96f2b106-9788-47a4-8f55-9609724a1fa0)